### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by pet3r-pan** [WizardOfAus/Brain-Computer-Interfaces-Starter-Pack#1]
This looks amazing... I have actually been meaning to dig into BCI as well. Got telegram or smthng? Would love to connect

**Comment by WizardOfAus** [comment]
@pet3r-pan yes would love to! Especially can learn together :) I will send you a telegram msg
